backend/deepcell_label/blueprints.py

```diff
@@ -75,7 +75,9 @@ def create_project():
         )
     labels_url = request.form['labels'] if 'labels' in request.form else None
     axes = request.form['axes'] if 'axes' in request.form else None
-    with tempfile.NamedTemporaryFile(delete=DELETE_TEMP) as image_file, tempfile.NamedTemporaryFile(delete=DELETE_TEMP) as label_file:
+    with tempfile.NamedTemporaryFile(
+        delete=DELETE_TEMP
+    ) as image_file, tempfile.NamedTemporaryFile(delete=DELETE_TEMP) as label_file:
```

Comment:
I thought the operating system changes were introduced in another PR. Is this commit left over from old branch history or were there some changes that didn't make it into the last PR?

Reply:
This is left over from old history; I think my branch was not up to date, sorry